### ISSUE TITLE
CURLFile breaks old PHP + autoload

### ISSUE THREAD

**Issue by remistr** [stripe/stripe-php#132]
If you are on an older version of PHP like 5.3 and have an autoloader then our bindings break trying to find `CURLFile` with the following error:

```
include(CURLFile.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory
```

The error seems to be linked to this commit: https://github.com/stripe/stripe-php/commit/70d35992dcf77a9886f90f5315a3c4ba7570759a which doesn't take the autoloader into account.

**Comment by noinput** [comment]
Running PHP 5.4.36 and getting the same here. Rolled back to 1.17.3 and back in business.

**Comment by bkrausz** [comment]
Fixed by ba06d1b44157e4fd53a572dc78008f34572bf665, will push a new release once tests pass.
